### PR TITLE
add missing @override

### PR DESCRIPTION
**Pull request by grimreaper**

### CODE REVIEW

**Comment by thpr** [comment]
So this one I would challenge as well - it's derived code, so we need some thought on modifying it

**Comment by grimreaper** [comment]
ack. Can we mark the derived/generated code as such? That way we could exclude it from checkstyle easily as well.

**Comment by thpr** [comment]
I think this is the only file that is mixed generated/ungenerated.  Anything that is auto generated that we don't customize we don't have in git.  Javacc will just generate it on demand.  We had to do that for the latter parts of this particular file.  You'll note a comment in the file that reads thus:

Items below this point are not auto-generated by javacc, but content unique to this implementation.

Basically everything above that point was auto-generated.  (The first line of the file also indicates that)
